external/storm-cassandra/src/test/java/org/apache/storm/cassandra/DynamicStatementBuilderTest.java

```diff
@@ -116,15 +124,15 @@ private void executeStatementAndAssert(CQLStatementTupleMapper mapper, String...
         List<Statement> map = mapper.map(Maps.newHashMap(), cassandraCQLUnit.session, mockTuple);
 
         List<String> listExpected = Arrays.asList(expected);
-        for( int i=0; i< map.size(); i++) {
+        for (int i = 0; i < map.size(); i++) {
             Assert.assertEquals(listExpected.get(i), map.get(i).toString());
         }
     }
 
     @Test
     public void shouldBuildStaticBoundStatement() {
-        CQLStatementTupleMapper mapper = boundQuery("INSERT INTO weather.temperature(weatherstation_id, event_time, temperature) VALUES(?, ?, ?)")
-                .bind(with(field("weatherstation_id"), field("event_time").now(), field("temperature")));
+        CQLStatementTupleMapper mapper = boundQuery("INSERT INTO weather.temperature(weather_station_id, event_time, temperature) VALUES(?, ?, ?)")
+                .bind(with(field("weather_station_id"), field("event_time").now(), field("temperature")));
```

Comment:
Cleaning up these test cases is part of another JIRA STORM-1348. The above changes are done as part of refactoring the cassandra table script.